README.md

```diff
@@ -53,12 +53,12 @@ docker run --network host sniff python packet_sniffer.py --help
 ```
 
 Usage of `--network host` is not supported on OS X or Windows
-so this container won't be fully functional - but you will see packets 
+so this container won't be fully functional - but you will see packets
 traveling within the docker subnet.
 
 ## Usage
 
-```
+```sh
```

Comment:
This marker too.

Reply:
Made a new commit with the required changes

Reply:
Done.